### DISCUSSION TITLE
How to truncate text in table cell, to avoid multi lines?

### DISCUSSION THREAD

**Discussion by mhasior** [QuestPDF/QuestPDF#395] [Q&A] [answered]
I need to keep all rows in a table to be single line, so I'd like to truncate text to available space. 
It would be also nice if: 
* the full text would be still available for copying
* there wold be some indicator that text was truncated - (...) or so

**Comment by DawidRobertNowak** [comment, marked as answer]
Hello,

We are closing this thread due to inactivity. We apologize for the delayed response. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

The functionality you described is now available through the [Clamp Line with Ellipsis](https://www.questpdf.com/api-reference/text/paragraph-style.html#clamp-line-with-ellipsis) paragraph style. You can use `.ClampLines(1)` to limit text to a single line with an automatic ellipsis indicator when truncated.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
